core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -170,7 +170,13 @@
 
   <div ng-if="isOnTerminalCard() && isLoggedIn && !isInPreviewMode && !isIframed && isCurrentCardAtEndOfTranscript()"
        class="conversation-skin-final-ratings">
-
+    <div style="color: #eee; margin-bottom: 20px;" translate="I18N_PLAYER_SHARE_EXPLORATION">
+    </div>
+      <div flex="55">
+        <sharing-links flex="50" layout-type="row" layout-align-type="end center"
+                         twitter-text="DEFAULT_TWITTER_SHARE_MESSAGE_PLAYER">
```

Comment:
try and match the leftmost character of "twitter-text" with the leftmost character of "flex" in the line above